src/PointwiseFunctions/Elasticity/PotentialEnergy.hpp

```diff
@@ -0,0 +1,52 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/Systems/Elasticity/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"
+
+namespace Elasticity {
+/*!
+ * \brief The potential energy density \f$U=-\frac{1}{2}S_{ij}T^{ij}\f$ stored
+ * in the deformation of the elastic material (see Eq. 11.25 in \cite
+ * ThorneBlandford2017)
```

Comment:
Have you tried rendering the docs? I think the line break after `\cite` doesn't work.